src/towncrier/test/helpers.py

```diff
@@ -1,22 +1,16 @@
 from __future__ import annotations
 
-import sys
 import textwrap
 
 from functools import wraps
+from importlib import resources
 from pathlib import Path
 from subprocess import call
 from typing import Any, Callable
 
 from click.testing import CliRunner
 
 
-if sys.version_info < (3, 9):
```

Comment:
I was looking only into this import ... but i see that we also have imports for `if sys.version_info < (3, 10):`

So importlib_resources can be removed only after we remove support for Python 3.9